Makefile

```diff
@@ -87,6 +87,10 @@ else
 ifeq ($(CONFIG_BUILD_LOADABLE),)
 
 $(BIN): $(foreach SDIR, $(CONFIGURED_APPS), $(SDIR)_all)
+	$(RM) $(BIN)
```

Comment:
Is this to remove old members that are not going to get replaced with the 'r' of `ar crs`?

Reply:
Yes